### PR TITLE
Collections: Remove selected categories layout

### PR DESCRIPTION
**Pull request by ninarg**
# Why?

We used to have a special layout which we applied to only a few selected collections. It has been decided to drop support for this, and rather display all collections the same way. 

# What?

Delete `.selectedCategories` section, and associated cell.

# Version Change

Breaking.

# UI Changes

The layout with boxes and icon isn't supported anymore, now it looks like a regular collection.

| Before  | After |
| --- | --- |
| IMG_0178 | simulator_screenshot_41859EFC-961B-45F0-A55F-1F971F084DD0 |

### CODE REVIEW

File: Sources/Fullscreen/ExploreDetail/ExploreDetailView.swift [hunk 1]
```diff
@@ -59,6 +59,7 @@ public final class ExploreDetailView: UIView {
     }
     private let imageCache = ImageMemoryCache()
     private let layoutBuilder = ExploreDetailLayoutBuilder(elementKind: UICollectionView.elementKindSectionHeader)
+    private let heroViewHeight: CGFloat = 220
```
Comment (BjornRuud) [inline Sources/Fullscreen/ExploreDetail/ExploreDetailView.swift:62]: Is the hero view content always such that it's okay to set it to a constant height?

**Review by BjornRuud — APPROVED** [review]
Nice cleanup!

**Review by AndreyMomot — APPROVED** [review]
Nice

File: Sources/Fullscreen/ExploreDetail/ExploreDetailView.swift (hunk 1 above)
Comment (ninarg) [reply Sources/Fullscreen/ExploreDetail/ExploreDetailView.swift:62]: yes, the content is pretty static, so the height has always been 220 👍

**Pull request merged**